tests/Avalonia.Controls.UnitTests/MaskedTextBoxTests.cs

```diff
@@ -559,7 +559,7 @@ public void SelectedText_Changes_OnSelectionChange()
                     Text = "0123456789"
                 };
 
-                Assert.True(target.SelectedText == "");
+                Assert.True(string.IsNullOrEmpty(target.SelectedText));
```

Comment:
I would say we want to validate that SelectedText/SearchText are not null, but empty in these controls.
Probably this analyzer is less useful for the tests.